MetaCocktailsSwiftData/Views/Search View/SearchCriteriaViewModel.swift

```diff
@@ -12,7 +12,7 @@ final class SearchCriteriaViewModel: ObservableObject {
     
     @Published var searchText: String = ""
     @Published var cocktailComponents = createComponentArray().sorted(by: { $0.name < $1.name })
```

Comment:
Lets make a stored array here like we did for GeneratedBoozeComponents(), no reason to have to run a function to loop over every Item in the array each time we search. We should just run in the first time and store it.